index.html

```diff
@@ -11,12 +11,12 @@
     <header>
         <a href="#">Ahsan Ghazni Khan</a>
         <nav>
-            <button><img src="images/Icon-Menu.png" alt="Ham burger button icon"></button>
-            <ul class="mnubr">
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#abtme">About</a></li>
-                <li><a href="#cntct">Contact</a></li>
-                <li class="wlt"><a href=""><img src="/images/wallet-icon-desktop.png" alt="Icon of the recieved messages"></a></li>
+            <button type="button" id="hmbtn"><img src="images/Icon-Menu.png" alt="Ham burger button icon"></button>
+            <ul class="mnubr" id="mnubr">
+                <li><a class="mnulinks" href="#">Portfolio</a></li>
+                <li><a class="mnulinks" href="#abtme">About</a></li>
```

Comment:
- [optional] Great job. you can make the mobile popup responsive so it disappears when switching to the desktop version. nice work

Reply:
yeah sure thanks